### ISSUE TITLE
Networking.sendRequest got 9 arguments, expected 8

### ISSUE THREAD

**Issue by rclai** [expo/expo#269]
When I scan my with the Expo app and the app loads, I get the red screen saying the above error. This is on the Android client.

I don't know if this has anything to do with upgrading RN to 0.45.1?

**Comment by brentvatne** [comment]
hello! yes it does, each version of react-native is matched with a specific expo sdk version: https://docs.expo.io/versions/v17.0.0/sdk/index.html#sdk-version -- you need to use 0.44.0 (or to keep things simple, our fork which is almost exactly the same: `"react-native": "https://github.com/expo/react-native/archive/sdk-17.0.0.tar.gz"`)

**Comment by brentvatne** [comment]
also -- if you have an idea for how to make this more clear to people in our docs or elsewhere, please let me know!

**Comment by rclai** [comment]
Oh thanks.

**Comment by ejoo** [comment]
I'm using 0.44.0 and I'm still getting that error.

**Comment by rey1024** [comment]
yes, i have same problem. Immediately after  run create-react-native-app, then i run yarn start or npm start. React Native #Networking.sendRequest got 9 arguments, expected 8

**Comment by brentvatne** [comment]
@ejoo @rey1024 - can you share your app.json and package.json here please?

**Comment by rey1024** [comment]
@brentvatne, this is my package.json
```
{
  "name": "AppBelajar",
  "version": "0.1.0",
  "private": true,
  "devDependencies": {
    "react-native-scripts": "0.0.40",
    "jest-expo": "~18.0.0",
    "react-test-renderer": "16.0.0-alpha.12"
  },
  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
  "scripts": {
    "start": "react-native-scripts start",
    "eject": "react-native-scripts eject",
    "android": "react-native-scripts android",
    "ios": "react-native-scripts ios",
    "test": "node node_modules/jest/bin/jest.js --watch"
  },
  "jest": {
    "preset": "jest-expo"
  },
  "dependencies": {
    "expo": "^18.0.3",
    "react": "16.0.0-alpha.12",
    "react-native": "^0.45.1"
  }
}
```

**Comment by rey1024** [comment]
[Solved] i upgrade react-native and create new project. 
https://facebook.github.io/react-native/docs/upgrading.html

**Comment by ejoo** [comment]
@brentvatne I've same JSON file as @rey1024

**Comment by dikaiosune** [comment]
@ejoo have you also tried updating the client app on your phone? If you have the same package.json as @rey1024 then it should work on the latest version of Expo.

**Comment by yash2code** [comment]
got the same error ...whats the solution?

**Comment by ParidelPooya** [comment]
I have the same problem. which version of react and react-native I should use for Expo 18.0.3?

**Comment by purely-smile** [comment]
same error, how to resolve it. please help me.

**Comment by ide** [comment]
This might have to do with a mismatch between the native code and your JavaScript bundle. These steps might help: https://forums.expo.io/t/how-to-clear-the-react-native-packager/1352/2.

**Comment by brentvatne** [comment]
indeed, make sure you update the react-native version in package.json to be compatible with the sdkVersion in app.json. see release notes for whatever version you're using at blog.expo.io

**Comment by SupriyaKalghatgi** [comment]
> also -- if you have an idea for how to make this more clear to people in our docs or elsewhere, please let me know!

@brentvatne Can we have these version compatibility details either in ReadMe on GitHub or with the Release Notes?

**Comment by ide** [comment]
The release notes (https://blog.expo.io/expo-sdk-v19-0-0-is-now-available-821a62b58d3d) talk about this, see the "Upgrading your app" section at the end.

**Comment by SupriyaKalghatgi** [comment]
@ide Yes i know that, but it would be more easy to get this details from ReadMe / Release Notes as mentioned earlier
